components/DesktopInventoryItemList/index.tsx

```diff
@@ -172,35 +162,90 @@ interface Props {
   category: string
 }
 
+const DEFAULT_ROWS_PER_PAGE = 5
+const DEFAULT_ORDER_BY = 'name'
+const DEFAULT_ORDER = 'asc'
+
 export default function DesktopInventoryItemList(props: Props) {
-  const [order, setOrder] = React.useState<Order>('asc')
-  const [orderBy, setOrderBy] = React.useState<keyof Data>('name')
+  const [order, setOrder] = React.useState<Order>(DEFAULT_ORDER)
+  const [orderBy, setOrderBy] = React.useState<keyof Data>(DEFAULT_ORDER_BY)
   const [page, setPage] = React.useState(0)
-  const [rowsPerPage, setRowsPerPage] = React.useState(5)
+  const [rowsPerPage, setRowsPerPage] = React.useState(DEFAULT_ROWS_PER_PAGE)
+  const [visibleRows, setVisibleRows] = React.useState<Data[] | null>(null)
+  const [tableData, setTableData] = React.useState<Data[]>([])
```

Comment:
Type of `tableData` incompatible with `Data[]` when used below.

The error shows each field having type `string | number` but in `Data`, most types have `string` and a few have specialized types such as the `attributes` member.